### ISSUE TITLE
[Accessibility] Help icon in PR migrator do not scale well at high resolution and high DPI

### ISSUE THREAD

**Issue by PatoBeltran** [NuGet/Home#6830]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

VS version (if appropriate): 15.7 preview 4

## Detailed repro steps so we can see the same problem

1. Change the resolution to 1560 x 1440 or greater.
2. Change the scaling to 175%.
3. Launch VS and create new net framework project. (packages.config)
4. Install jquery v3.3.1 and Newtonsoft.Json v6.0.4
5. Right click on packages.config and select "Migrate packages.config to PackageReference"

...

## Expected
The help icon should be rendered without aliasing.

## Actual
The help icon do not have smooth edges -

![help icon](https://user-images.githubusercontent.com/2132567/38759517-2c6652e4-3f2b-11e8-82e3-10b18e92da1c.PNG)
